<?xml version="1.0" encoding="UTF-8"?>

<ui version="4.0">
 <class>ControllerConfigView</class>
 <widget class="QDialog" name="ControllerConfigView">
  <property name="windowTitle">
   <string>Controller Configuration</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>120</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>No Controllers Connected</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="countLabel">
     <property name="text">
      <string>Controllers Connected: 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Controller Name(s): â€”</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bluetoothButton">
      <property name="text">
        <string>Open Bluetooth Settings</string>
      </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>